pkg/controller/account/account_controller.go

```diff
@@ -167,13 +167,13 @@ func (r *ReconcileAccount) Reconcile(request reconcile.Request) (reconcile.Resul
 	}
 
 	// Log accounts that have failed and don't attempt to reconcile them
-	if accountIsFailed(currentAcctInstance) {
+	if currentAcctInstance.IsFailed() {
 		reqLogger.Info(fmt.Sprintf("Account %s is failed. Ignoring.", currentAcctInstance.Name))
 		return reconcile.Result{}, nil
 	}
 
 	// Detect accounts for which we kicked off asynchronous region initialization
-	if accountIsInitializingRegions(currentAcctInstance) {
+	if currentAcctInstance.IsInitializingRegions() {
 		// Detect whether we set the InitializingRegions condition in *this* invocation of the
 		// operator or a previous one.
 		if regionInitStale(currentAcctInstance) {
```

Comment:
This (`regionInitStale`) could also be method-ified.

Reply:
At first pass I tried that as well, but the problem lies with trying to import `utils` to get the operator start time, because `utils` imports `awsv1alpha1` and when you try to get the operator start time from utils it causes a cyclical dependency.  If you have a suggestion as to how to get around that without another relatively major refactor, then I'd be happy to try it.

Reply:
Bleh, yeah, that sucks. I played with it a little bit too, but there's no easy answer. It would be nice to add a TODO to the method to figure it out for the future, but meh.

Reply:
We _could_ pull out the operator startup thing into its own package, which should work.  Or if you add that to the bootstrap package you're working on we could use that instead?  Or we could propose that to the upstream maybe?

Reply:
I suspect it's too hacky for upstream, but maybe worth a shot.

An isolated package could work, though again kind of hacky.

We're *far* away from having actual go code in boilerplate.